IBPSA/Utilities/Math/Functions/regNonZeroPower.mo

```diff
@@ -4,7 +4,9 @@ function regNonZeroPower
   extends Modelica.Icons.Function;
 
  input Real x "Abscissa value";
- input Real n "Exponent";
+ input Real n(
+   min=0.000001,
```

Comment:
Would Modelica.Constants.eps reach a similar goal like this small number?

Reply:
I first had Modelica.Constants.eps, but this may be too small so I hard coded a small number.